core/src/main/java/io/substrait/expression/WindowBound.java

```diff
@@ -45,7 +45,7 @@ public BoundedKind boundedKind() {
 
     public abstract Direction direction();
 
-    public abstract Expression offset();
+    public abstract long offset();
```

Comment:
In the [spec](https://github.com/substrait-io/substrait/blob/51765cccd92fa0014e9cf077cc8cc952731f33d0/proto/substrait/algebra.proto#L933C5-L948), offset must be `int64`.

The POJO layer should reflect this restriction.